QMigrate/signal/trigger.py

```diff
@@ -173,7 +176,7 @@ def __init__(self, lut, run_path, run_name, **kwargs):
         else:
             raise util.InvalidThresholdMethodException
         self.marginal_window = kwargs.get("marginal_window", 2.)
-        self.minimum_repeat = kwargs.get("minimum_repeat", 4.)
+        self.min_event_interval = kwargs.get("min_event_interval", 4.)
```

Comment:
```suggestion
        self.min_event_interval = kwargs.get("min_event_interval", 4.)
        self.minimum_repeat = kwargs.get("minimum_repeat", 4.)
```

I doubt anyone is setting the minimum repeat in this way, but would need to include to have completely coverage.

Reply:
Nice catch.